src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const objCalc = {
```

Comment:
```suggestion
  const calculator = {
```
Would be better. Adding the word `obj` does not improve understanding of what this variable is.